src/Twig/RuntimeExtension.php

```diff
@@ -85,16 +85,12 @@ public function getMenu(Request $request): ?array
     public function getNotifications(): ?NotificationEvent
     {
         if (!$this->eventDispatcher->hasListeners(NotificationEvent::class)) {
-            return null;
+            return new NotificationEvent();
```

Comment:
ups, missed that, but: can this method still return null?

Reply:
As `{% if notifications is not null %}` was removed form the template, **NO**.
And you asked for that: https://github.com/kevinpapst/TablerBundle/pull/176#discussion_r1381603445 😄

Reply:
Hahaha, and are you going to change it?

Reply:
I am talking about

```php
    public function getNotifications(): ?NotificationEvent
```

vs

```php
    public function getNotifications(): NotificationEvent
```

Reply:
Oh yes sure!
(didn't see you edited message above)